kolibri/content/management/commands/importcontent.py

```diff
@@ -64,35 +64,39 @@ def download_content(self, channel_id):
     def copy_content(self, channel_id, path):
         self._transfer(COPY_METHOD, channel_id, path=path)
 
-    def _transfer(self, method, channel_id, path=None):
+    def _transfer(self, method, channel_id, path=None):  # noqa: max-complexity=16
```

Comment:
If we have to do this, it might be better to just break up this function into subfunctions instead.

But eh, we'll be porting this to a better way once the new version of iceqube comes in, so not urgent.

Reply:
Yeah, this was a manana comment.